src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/signature_approved.jelly

```diff
@@ -0,0 +1,70 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="custom-pane-frame-padding">
```

Comment:
This should have the `.pane-frame` before the `.custom-pane-frame-padding` class. Reasons:

- Many styles are applied to `.bigtable` and `.pane` with selectors nested under `.pane-frame`.
- It will make the table styles not be able to be updated when the tables are restyled on Jenkins side.